kong/pdk/request.lua

```diff
@@ -619,14 +617,14 @@ local function new(self)
   -- kong.request.get_header("Host")            -- "foo.com"
   -- kong.request.get_header("x-custom-header") -- "bla"
   -- kong.request.get_header("X-Another")       -- "foo bar"
-  function _REQUEST.get_header(name)
+  function _REQUEST.get_header(name, ctx)
```

Comment:
`http_get_header` supports `ctx` param to cache headers, expose it to `get_header` API so we could use cache.